<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>341</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>51</y>
      <width>341</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>580</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>reconocer</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>520</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>seleccione un metodo de extracci√≥n</string>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>20</y>
       <width>311</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>520</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Seleccionar imagen de entrada</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>seleccionar</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>580</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Segmentar imagen en letras</string>
    </property>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Segmentar</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>50</y>
      <width>341</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>Texto reconocido</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>291</width>
       <height>191</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lohit Tamil</family>
       <pointsize>20</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>310</y>
       <width>291</width>
       <height>191</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lohit Tamil</family>
       <pointsize>20</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>METODO K-GABOR</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>GABOR CONVENCIONAL</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1154</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadimage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>587</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>segment()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>recognition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>631</y>
    </hint>
    <hint type="destinationlabel">
     <x>659</x>
     <y>615</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>segment()</slot>
  <slot>recognition()</slot>
 </slots>
</ui>
